core/src/com/unciv/logic/map/tile/TileImprovementFunctions.kt

```diff
@@ -311,31 +311,32 @@ class TileImprovementFunctions(val tile: Tile) {
         val closestCity = civ.cities.minByOrNull { it.getCenterTile().aerialDistanceTo(tile) }
             ?: return
         val distance = closestCity.getCenterTile().aerialDistanceTo(tile)
-        var productionPoints = 0f
+        var stats = Stats()
         for (unique in tile.getTerrainMatchingUniques(UniqueType.ProductionBonusWhenRemoved)) {
-            productionPoints = unique.params[0].toFloatOrNull() ?: return
+            //productionPoints = unique.params[0].toFloatOrNull() ?: return
+            stats.add(unique.stats)
         }
         val ruleset = civ.gameInfo.ruleset
         val choppingYieldsIncreaseWithGameProgress = ruleset.modOptions.constants.choppingYieldsIncreaseWithGameProgress
         // Civ6 yields increase with game progression: https://www.reddit.com/r/civ/comments/gvx44v/comment/fsrifc2/
         if (choppingYieldsIncreaseWithGameProgress) {
             val gameProgress = max(civ.tech.researchedTechnologies.size.toFloat() / ruleset.technologies.size, civ.policies.adoptedPolicies.size.toFloat() / ruleset.policies.size)
-            productionPoints *= (1 + 9 * gameProgress)
+            stats *= (1 + 9 * gameProgress)
+        }
+        if (distance != 1) stats *= (6-distance)/4
+        if (tile.owningCity == null || tile.owningCity!!.civ != civ) stats *= 2/3
```

Comment:
2/3 is with ints, so that boils down to '1'

Also these checks can be combined but I'm not sure if it's more readable or less
```suggestion
        if (tile.owningCity == null || tile.owningCity!!.civ != civ) stats *= 2/3f
```